linux_files/setup

```diff
@@ -775,11 +775,13 @@ function dockerinstall_build_relay {
         fi
 
         GOOS=windows go build -o npiperelay.exe github.com/jstarks/npiperelay
-        sudo mkdir -p ${wHome}/.npiperelay
+        sudo mkdir -p "${wHome}/.npiperelay"
         cmd.exe /c 'attrib +h %HOMEDRIVE%%HOMEPATH%\.npiperelay'
-        sudo cp npiperelay.exe ${wHome}/.npiperelay/npiperelay.exe
+        sudo cp npiperelay.exe "${wHome}/.npiperelay/npiperelay.exe"
     fi
 
+    ln -s "${wHome}/.npiperelay/npiperelay.exe" /usr/local/bin/npiperelay.exe
```

Comment:
Please delete the symbolic link. It was deleted due to problems when the root is changed. This is why the relay always ask for the latest home directory.